README.md

```diff
@@ -1,3 +1,88 @@
-# TODO Repository Name
+# Knative Monitoring
 
-_TODO(leads): please fill in this section_
+This repository contains Grafana Dashboards and Prometheus Scraping configurations to monitor Knative.
+
+You will need to deploy the following ServiceMonitors.
+
+Also, kube-state-metrics v2+ doesn't export all labels by default which means you must pass the following argument to kube-state-metrics:
+
+```
+- --metric-labels-allowlist=pods=[*],deployments=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance]
+```
+Once kube-state-metrics has been adjusted, you will need to deploy the following servicemonitors and import the dashboards in the grafana folder.
```

Comment:
> kube-state-metrics v2+

Is this a pre-req we need to install? If so can we add it to a pre-req section and link out

Is this coupled with any version of K8s?  I ask because for Knative 0.24 our min K8s version will be 1.19.

Reply:
nm - I see the compatibility matrix https://github.com/kubernetes/kube-state-metrics#compatibility-matrix that it shows v2 supports k8s 1.19

Reply:
It is a pre req for one of the dashboards.

I tested it with my prod home cluster RKE on VMWare running 1.19.6 and a GKE Cluster running 1.20 running 16.9.1 of the prom stack